src/main/java/io/lettuce/core/metrics/MicrometerOptions.java

```diff
@@ -234,6 +246,18 @@ public Builder targetPercentiles(double[] targetPercentiles) {
             return this;
         }
 
+        /**
+         * Sets which commands are enabled for latency recording. Defaults to an empty list, which means all commands will be recorded
+         * See {@link MicrometerOptions#DEFAULT_ENABLED_COMMANDS}.
+         *
+         * @param commands list of Redis commands that are enabled for latency recording
+         * @return this {@link Builder}.
+         */
+        public Builder enabledCommands(CommandType... commands) {
```

Comment:
How about having a `Predicate<RedisCommand>` instead of protocol keywords? We could provide derived methods that can build a predicate to filter unwanted commands from being recorded?